tests/InjectorTest.php

```diff
@@ -550,6 +551,8 @@ public function testInvokeable(): void
         $this->assertSame(42, $result);
     }
 
+    // Injector::make tests
```

Comment:
How about having two separate files? One for make, another for invoke? Would that be better in terms of readability?